packages/js/number/src/index.ts

```diff
@@ -29,8 +41,7 @@ export function numberFormat(
 		return '';
 	}
 
-	let parsedPrecision = parseInt( precision, 10 );
-
+	let parsedPrecision = precision === null ? NaN : Number( precision );
```

Comment:
Updated this line because the first argument of`parseInt` only accepts the string type.

```ts
function parseInt(s: string, radix?: number | undefined): number
```

Reply:
Number(_) and parseInt(, 10) behave differently when the input is a string that starts with 0x but I reckon that's not relevant here as I don't think anyone would have been trying to pass in precision using non-decimal numbers :laugh